utils/dockerutil.py

```diff
@@ -392,6 +392,14 @@ def image_tag_extractor(cls, entity, key):
                     tags.add(split[key])
             if len(tags) > 0:
                 return list(tags)
+        elif entity.get('RepoDigests'):
+            # the human-readable tag is not mentioned in RepoDigests
+            if key > 0:
```

Comment:
key is lower or equal to 0 (it's `== 0` actually). I'll update to `!=` for clarity